Email.Service/Email.Service/Program.cs

```diff
@@ -0,0 +1,63 @@
+using Email.Service.BLL.Consumers;
+using Email.Service.BLL.DI;
+using Email.Service.Helper;
+using MassTransit;
+
+namespace Email.Service;
+
+public class Program
+{
+    public static void Main(string[] args)
+    {
+        var builder = WebApplication.CreateBuilder(args);
+
+        builder.Services.Configure<EmailSettings>(builder.Configuration.GetSection("EmailSettings"));
```

Comment:
i'm not sure in which layer it should be configured, probabably, in bll as well
So, better find out it